.drone.yml

```diff
@@ -13,15 +10,16 @@ notify:
     webhook_url: $$slack_webhook
 publish:
   npm:
-    email: $$email
-    password: $$npmdeploypassword
-    username: $$npmdeployusername
+    email: $$npm_drone_email
+    password: $$npm_drone_password
+    registry: https://registry.npmjs.org
+    username: $$npm_drone_username
     when:
       branch: master
 script:
-- sudo pip install -q git+https://$REPORT_CARD_GITHUB_REPO_TOKEN@github.com/Clever/report-card.git; GITHUB_API_TOKEN=$REPORT_CARD_GITHUB_STATUS_TOKEN report-card --publish || true
+- nvm install 0.10.29
+- npm install npm@2.7.0 -g
 - npm config set ca ""
```

Comment:
i think it may be necessary since this is still based off the node 0.10 docker image. i think i'll leave it for now. we can update that later.